docs/user-manual/en/broker-balancers.md

```diff
@@ -0,0 +1,176 @@
+# Broker Balancers
+Apache ActiveMQ Artemis broker balancers allow incoming client connections to be distributed across multiple [target brokers](target-brokers).
+The target brokers are grouped in [pools](#pools) and the broker balancers use a [target key](#target-key)
+to select a target broker from a pool of brokers according to a [policy](#policies).
+
+### This feature is still **EXPERIMENTAL** and not meant to be run in production yet. Furthermore, its configuration can change until declared as **officially stable**.
+
+## Target Broker
+Target broker is a broker that can accept incoming client connections and is local or remote.
+The local target is a special target that represents the same broker hosting the broker balancer.
+The remote target is another reachable broker.
+
+## Target Key
+The broker balancer uses a target key to select a target broker.
+It is a string retrieved from an incoming client connection, the supported values are:
+* `CLIENT_ID` is the JMS client ID;
+* `SNI_HOST` is the hostname indicated by the client in the SNI extension of the TLS protocol;
+* `SOURCE_IP` is the source IP address of the client;
+* `USER_NAME` is the username indicated by the client.
+
+## Pools
+The pool is a group of target brokers and checks periodically their state.
+It provides a list of ready target brokers to distribute incoming client connections only when it is active.
+A pool becomes active when the minimum number of ready target brokers defined by the `quorum-size` parameter is reached.
+When it is not active, it doesn't provide any target avoiding weird distribution at startup or after a restart.
+Including the local target broker in the pool allows broker hosting the broker balancer to accept incoming client connections. 
+By default, a pool doesn't include the local target broker, to include it the `local-target-enabled` parameter must be `true`.
```

Comment:
```suggestion
By default, a pool doesn't include the local broker, to include it as a target the `local-target-enabled` parameter must be `true`.
```
Moving 'target' for clarity, as it isnt a 'target broker' yet if it isnt in the pool, which is whats explained next.